var/spack/repos/builtin/packages/paddle/package.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class Paddle(CMakePackage):
+    """Parallel algebraic domain decomposition for linear algebra software package."""
+
+    homepage = "https://gitlab.inria.fr/solverstack/paddle"
+    git      = "https://gitlab.inria.fr/solverstack/paddle.git"
+    maintainers("fpruvost")
+
+    version("0.3.7", tag="0.3.7", submodules=True)
+    version('master', branch='master', submodules=True)
+
+    # cmake's specific
+    variant("shared", default=True, description="Build a shared library")
+
+    # paddle's specific
+    variant("parmetis", default=False, description="Enable ParMETIS ordering")
+    variant("tests", default=False, description='Enable compilation and installation of testing executables')
+
+    depends_on("mpi")
+    depends_on("scotch~metis+mpi")
+    depends_on("parmetis",when="+parmetis")
+
+    root_cmakelists_dir = 'src'
+
+    def cmake_args(self):
+        spec = self.spec
+
+        args = []
+
+        args.extend([
+            "-Wno-dev",
+            "-DCMAKE_COLOR_MAKEFILE:BOOL=ON",
+            "-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON",
+            "-DCMAKE_BUILD_TYPE=%s"         % ('Debug' if '+debug'   in spec else 'Release'),
+            "-DBUILD_SHARED_LIBS=%s"        % ('ON'    if '+shared'  in spec else 'OFF'),
+            "-DPADDLE_BUILD_TESTS=%s"       % ('ON'    if '+tests'   in spec else 'OFF'),
+            "-DPADDLE_ORDERING_PARMETIS=%s" % ('ON'    if '+parmets' in spec else 'OFF')
```

Comment:
Typo, but I do recommend using the helper method for this.

```suggestion
            "-DPADDLE_ORDERING_PARMETIS=%s" % ('ON'    if '+parmetis' in spec else 'OFF')
```